weblate/addons/tasks.py

```diff
@@ -112,6 +115,8 @@ def language_consistency(
         )
     ).exclude(translation_count=languages.count())
 
+    log_result = ""
```

Comment:
The repeated string concatenation will perform badly with a log of log entries, use list and merge it with join in the end:

```suggestion
    log_result : list[str] = []
```